src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -64,7 +64,7 @@
     <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
     <PackageReference Include="FirebirdSql.Data.FirebirdClient" Version="8.5.2" />
-    <PackageReference Include="Npgsql" Version="4.1.9" />
+    <PackageReference Include="Npgsql" Version="5.0.11" />
```

Comment:
The whole list of dependencies was in conflict, but the only significant change from 5.3.x I have seen was this one.

Reply:
getting AsyncNotDisposed error with Nhibernate upgrade can you help?